src/validations/rules/ssp.sch

```diff
@@ -1608,6 +1608,50 @@
                 test="current() = $security-objective-levels">[Section B Check 3.10] A FedRAMP SSP must specify an allowed security objective
                 value.</sch:assert>
         </sch:rule>
+        <sch:rule
+            context="o:control-implementation/o:implemented-requirement/o:statement/o:by-component/o:set-parameter"
```

Comment:
The context is too specific. `set-parameter` can occur as a child of `implemented-requirement` and `by-component` elements. It can also be found as a child of `control-implementation` (but consider that in a separate issue).

A context of `o:control-implementation/o:implemented-requirement/descendant-or-self::o:set-parameter` should suffice.